python/lsst/meas/algorithms/measureApCorr.py

```diff
@@ -177,7 +177,7 @@ def __init__(self, schema, **kwds):
         Task.__init__(self, **kwds)
         self.refFluxKeys = FluxKeys(self.config.refFluxName, schema)
         self.toCorrect = {}  # dict of flux field name prefix: FluxKeys instance
-        for name in getApCorrNameSet():
+        for name in sorted(list(getApCorrNameSet())):
```

Comment:
`sorted(apCorrNameSet())` is all you need; `sorted` can take any iterable, and always returns a new `list`.

Reply:
Sounds good; PR approved.